win32ss/user/ntuser/window.c

```diff
@@ -1511,8 +1511,13 @@ NtUserBuildHwndList(
    NTSTATUS Status;
    ULONG dwCount = 0;
 
+   UserEnterExclusive();
```

Comment:
Could be moved after the `pBufSize` check.

Reply:
OK.